new/LIT6190Malkea.xml

```diff
@@ -0,0 +1,69 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6190Malkea" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ እድና፡</title>
```

Comment:
I would mark up Edna (PRS3672edna) at some point, and/or add a relation
`<relation name="ecrm:P129_is_about" active="LIT6190Malkea" passive="PRS3672edna"/>`